### ISSUE TITLE
Improved Melting

### ISSUE THREAD

**Issue by mxnmnm** [FIREdog5/SereneTweaks#3]
Would it be possible to make the snow/ice in serene seasons melt just like in this mod? https://www.curseforge.com/minecraft/mc-mods/survival-inc (Also if you add this make sure that the snow/ice is only on the last stage of melting if the biome's default temperature always allows for snow/ice) If you need more info let me know!

**Comment by FIREdog5** [comment]
It looks like this mod adds it's own seasons as well, and doesn't use the seasons from serene seasons. I suspect those mods are incompatible, although I haven't tested that.. either way, if the author to survival inc provides an API, update me and I might look into it.

**Comment by mxnmnm** [comment]
Yea I know it adds its own seasons and such (they overlap and break when combined with serene seasons), I was thinking something along the lines of having different melting stages implemented in a similar way, maybe with the same textures and such. It is under MIT License so I don't think you'd need to worry about permission and such that much.

**Comment by FIREdog5** [comment]
Ok, this is a cool idea. It would require a bit of a rewrite to Minecraft weather, but It's given me a cool idea (similar I hope to what you want). I might do this after the 1.16 update (looking like I will probably just jump strait to 1.17 tho).

**Comment by mxnmnm** [comment]
I don't think it would need a rewrite to normal weather, it would probably just need to change how serene seasons melts stuff when the seasons change. Is there a way that this could be in 1.12? I would be happy to try to help (I know a bit of java but I'm not sure if I know enough to mod lots of things yet, so you might need to help me a bit, otherwise I'm willing to do what I can)
